### ISSUE TITLE
Random.py little things to fix

### ISSUE THREAD

**Issue by Harvester62** [lilly1987/AI-WEBUI-scripts-Random#3]
A few things I noticed in the Random.py execution at startup of the webui-user.bat that might need a check:

in the console the following messages are printed that weren't displayed before (a little confusing; are they really needed to be pinted on screen? Just asking for curiosity):

is_img2img : False;
is_img2img : True;

the following warnings are also displayed in the console, twice:

Warning: Bad ui setting value: customscript/Random.py/txt2img/fix width height direction/value: 0; Default value "0" will be used instead.
Warning: Bad ui setting value: customscript/Random.py/img2img/fix width height direction/value: 0; Default value "0" will be used instead.

At line 77 there is a misspelled word, "olny" instead of "only"

In the Web UI, the titles "olny i2i option" and "size" have a large spacing between characters; is it intentional? They look a little strange: "o l n y   i 2 i   o p t i o n" and "s i z e".

For better clarity of they functioning, may I suggest you to write the parameter's labels as follow, so it is clearer that they may work in both directions? It's only a suggestion.

E.g.:

steps min/max
steps max/min

width 1 min/max
width 2 max/min

and so on for all the other parameters.

Thank you.

**Comment by Harvester62** [comment]
To explain better why I was asking about the spacing between characters in the two above labels (only i2i option, and size) is because written in that way, in the localization json file they are considered as separate labels, and this is a problem because, for example, the letter "n" is already in use for other parameters a "number". To show you what I mean, please see the following screenshot (ah, sorry, I forgot to tell you that I am the translator for the Italian localization of the SD Web UI, Scripts and Extensions):

![immagine](https://user-images.githubusercontent.com/29801031/202894938-30b9eaee-58a5-4e78-847f-bcf8d6ced875.png)


I did notice this when translating the new parameters in your script, and wished to let you know. So, I would suggest you to do not separate each character in the above two phrases, if possible. Thank you for you attention.

**Comment by holy1017** [comment]
Temporarily answer in Korean
임시로 한국어로 답변.

. 불필요한 로그는 제거하겠다.

. gr.HighlightedText 가 문제를 발생시킨다. 일단 다른것으로 교체하겠다.

지금 일하는 중이라 수정은 저녁 이후에 하겠다. ^^

**Comment by Harvester62** [comment]
회신해 주셔서 감사합니다. 이 작은 문제를 해결하는 대로 회신을 기다리겠습니다. 여보세요.
Hope to listen from you soon and have a great day. Bye.

**Comment by lilly1987** [comment]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/828438b4a190759807f9054932cae3a8b880ddf1

최신 버전 기준으로 반영 완료
Reflected based on the latest version

-----
if step1 > step2 then rand(step2 , step1)
if step2 > step1 then rand(step1 , step2)

즉 두개값을 거꾸로 설정해도 알아서 값을 바궈서 넣어준다.
In other words, even if you set the two values oppositely, the values are reversed and entered.

적절한 라벨명이 생각나지 않는다.
I can't remember the proper label name.

**Comment by Harvester62** [comment]
Yes, that's how it works internally, but in the UI I think that it would make sense to show that min/max and max/min terminology because it let the user immediately understand that they can be set in reverse mode. 

Alternatively, you might simply name as:

parameter min
parameter max 

and add a comment or description pop-up when hovering over the parameters with the mouse, which explains that they would be inverted if set the contrary (max the first parameter and min the second), same message for all the parameters because the internal mechanism is the same for all of them. Just thinking how you might make it simpler.
